js/filelist.js

```diff
@@ -224,13 +224,14 @@ var Files_Linkeditor = {
 					window.context = context;
 					// Build the download url.
 					var downloadUrl = context.fileList.getDownloadUrl(filename);
-					// Find out where we are to use this link for the cancel button.
-					var currentUrl = encodeURI(context.fileList.linkTo() + '?path=' + context.dir);
 					// Find the element we are clicking on.
 					var linkElement = $('[href="'+ downloadUrl +'"]');
 					// Add a loading spinner.
 					linkElement.addClass('icon-loading');
 
+					// open a new window (or tab) directly to avoid the browsers popup warning
+					var linkWindow = window.open('./', '_blank');
```

Comment:
I think we can make the first parameter just `''` or `'about:blank'` instead, so that we don't load the page into memory a second time, before we're actually lazy loading the target page.

Reply:
definitely!
The browser actually does not know how to handle `'./'` as it opens the window in a completely new context, resulting in opening `about:blank` at least in firefox and chrome.